### ISSUE TITLE
Hotfix release available: 2018-04-22b "Greebo".

### ISSUE THREAD

**Issue by pagaille** [stephdl/nethserver-dokuwiki#1]
Unsure how to upgrade however. Doc invites you to download and copy / overwrite the tarball in the install directory, I guess /usr/share/dokuwiki/. Is it ok ?

**Comment by pagaille** [comment]
Bon, I did it the way they advice it in the doc. Looks like it's working except those warnings related to links : 

```
# 'cp' -af dokuwiki/* /usr/share/dokuwiki/                        
cp: cannot overwrite non-directory ‘/usr/share/dokuwiki/conf’ with directory ‘dokuwiki/conf’
cp: cannot overwrite non-directory ‘/usr/share/dokuwiki/data’ with directory ‘dokuwiki/data’
```

**Comment by stephdl** [comment]
dokuwiki released as 20180422b in repo, please upgrade from stephdl repo

Merci pour la remontée d'information
